pii_scan.py

```diff
@@ -54,6 +54,15 @@ def analyze_text(text: str, show_supported=False, show_details=False, score_thre
                                         patterns=[uuid_pattern])
     registry.add_recognizer(uuid_recognizer)
 
+
+    #Create an additional pattern to detect a 123456789 Student Id
+    student_id_pattern = Pattern(name='student_id',
+                           regex=r'\b[0-9a-fA-F][0-9a-fA-F][0-9a-fA-F][0-9a-fA-F][0-9a-fA-F][0-9a-fA-F][0-9a-fA-F][0-9a-fA-F][0-9a-fA-F]\b',
```

Comment:
In your issue description you say that a student ID is a 9 digit number

[0-9a-fA-F] will detect digits and letters, so, A23456789 would be detected as a STUDENT_ID which I don't think you want.

You probably want to do something like this which would detect only a digit 9 times
```
    student_id_pattern = Pattern(name='student_id',
                           regex=r'\b\d{9}\b',
                           score=0.8)
```